### ISSUE TITLE
Follow "action" URL inside the HTML forms

### ISSUE THREAD

**Issue by butlnor** [a85/POSTMan-Chrome-Extension#3016]
When the request's response returns a valid HTML, Postman can nicely render this HTML. 
If the HTML contains a &lt;form&gt; with some action URL and submit button, Postman doesn't follow this action URL when clicking submit. This would be a nice-to-have feature. 

As now I have to save the HTML file locally, open it in browser and click the submit button, but of course I cannot then get all the nice stuff and features that Postman provides, as the browser does what it does.

**Comment by shamasis** [comment]
I see that it is such a nice idea. But, presently, the preview of HTML is rendered in a sandbox mode to avoid security issues that is posed by running arbitrary code and URL navigation inside the HTML within preview tab. 

We will need to explore additional security aspects before we can think about this new feature. Or perhaps think about building a concept of isolated built-in browser. I am adding this as feature request and somewhat sure that it is not planned in short to mid term.

**Comment by bentcoder** [comment]
Could someone please add some kind of comment or label saying that **this feature won't be implemented** to prevent people wondering (_like myself_) what the state of this request is or plans etc? I can obviously see it is closed but why though!
